admin/src/components/Login/SignUp.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class SignUp extends React.PureComponent {
+  state = {
+    isLoading: false,
+    name: null,
+    email: null,
+    password: null
+  };
+
+  onSignUp(e) {
```

Comment:
Shouldn't that be split up into a container?

The SignUp component should only contain the UI code IMO and there should be a container that passes the state down and handles `onSubmit`.

Reply:
I think I disagree with separating into containers/components pre-maturely, its mostly adds indirection with no benefit IMO.

Reply:
I have thought about it a bit more, you are probably right creating the "reacty" container/component separation I think makes sense especially as the complexity of the components grows.